src/dataset-builder/ConceptSearch.ts

```diff
@@ -153,10 +153,13 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
         : spinnerOverlay,
     ]),
     cart.length !== 0 &&
-      h(ActionBar, {
-        prompt: cart.length === 1 ? '1 concept selected' : `${cart.length} concepts selected`,
-        actionText,
-        onClick: () => _.flow(onCommit)(cart),
-      }),
+      h(Fragment, [
+        div({ style: { width: '100%', height: ACTION_BAR_HEIGHT } }, []),
```

Comment:
I wasn't sure if this should get folded into the `ActionBar` component, but I was leaning to no. The reason for not moving it in is that right now, that component doesn't care where it is placed in the DOM tree, since it uses `fixed` positioning. If we add the blocking element, the action bar would need to care about where it is rendered in the DOM, because the spacer would get dropped wherever `ActionBar` is.